src/features/wisdom/inlineSuggestions.ts

```diff
@@ -94,6 +94,14 @@ async function getInlineSuggestionItems(
     inlineSuggestionDisplayed = false;
     return [];
   }
+
+  if (!wisdomSetting.basePath) {
+    vscode.window.showErrorMessage(
+      "Base path for wisdom service is empty. Please provide a base path"
```

Comment:
please change `wisdom` to `Project Wisdom`

Reply:
done